.vscode/settings.json

```diff
@@ -12,7 +12,7 @@
   "editor.formatOnSave": true,
   "editor.formatOnPaste": true,
   "editor.codeActionsOnSave": {
-    "source.organizeImports": false,
-    "source.removeUnusedImports": true
+    "source.organizeImports": "never",
+    "source.removeUnusedImports": "explicit"
```

Comment:
i think we should update this file, my vscode started to change it too, but i m not sure if it was mine or is a generic change.

Reply:
generic